### PR TITLE
fix(deps): add @types/cypress

### PR DESCRIPTION
**Pull request by diminutivesloop**
The Cypress typings are referenced in `cypress\tsconfig.json` but they're not installed as a dependency so editors like VSCode show errors when opening the test files.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,13 +47,16 @@
     "@types/bcryptjs": "2.4.2",
     "@types/bluebird": "3.5.33",
     "@types/connect-flash": "0.0.36",
+    "@types/connect-history-api-fallback": "1.3.3",
     "@types/cors": "2.8.9",
+    "@types/cypress": "^1.1.3",
```
Comment (kevinold) [inline package.json:52]: @diminutivesloop Thanks for your contribution! Would you pin this version to 1.1.3. (remove the ^)

**Review by kevinold — CHANGES_REQUESTED** [review]
Thanks for your contribution!  One minor edit and we will merge.

**Comment by bahmutov** [comment]
What’s types/cypress for?

**Comment by kevinold** [comment]
Actually, this package is deprecated -- see https://www.npmjs.com/package/@types/cypress and we maintain our own types in Cypress itself. See https://github.com/cypress-io/cypress/tree/develop/cli/types

**Comment by diminutivesloop** [comment]
@kevinold Yeah I actually noticed this when I set up Cypress in my own project. Any idea why the built-in types weren't being picked up by VSCode for this project?

**Comment by kevinold** [comment]
@diminutivesloop I'm not sure.  @bahmutov might have ideas.

**Comment by bahmutov** [comment]
@diminutivesloop having global types is always problematic, read https://glebbahmutov.com/blog/local-cypress/

I personally just use JavaScript with `/// <reference types="cypress" />`

**Pull request closed without merging**